src/Execution/MutationExecutor.php

```diff
@@ -235,6 +249,113 @@ public static function executeUpdate(Model $model, Collection $args, ?HasMany $p
+                }
+            });
+        });
+
+        $morphMany->each(function (array $nestedOperations, string $relationName) use ($model): void {
+            /** @var \Illuminate\Database\Eloquent\Relations\MorphMany $relation */
+            $relation = $model->{$relationName}();
+
+            collect($nestedOperations)->each(function ($values, string $operationKey) use ($relation): void {
+                if ($operationKey === 'create') {
+                    self::handleMultiRelationCreate(collect($values), $relation);
+                }
+                if ($operationKey === 'update') {
+                    collect($values)->each(function ($singleValues) use ($relation): void {
+                        self::executeUpdate($relation->getModel()->newInstance(), collect($singleValues), $relation);
+                    });
+                }
+
+                if ($operationKey === 'delete') {
+                    $relation->getModel()::destroy($values);
+                }
+            });
+        });
+
+        $morphOne->each(function (array $nestedOperations, string $relationName) use ($model): void {
+            /** @var \Illuminate\Database\Eloquent\Relations\MorphOne $relation */
+            $relation = $model->{$relationName}();
+
+            collect($nestedOperations)->each(function ($values, string $operationKey) use ($relation): void {
+                if ($operationKey === 'create') {
+                    self::handleSingleRelationCreate(collect($values), $relation);
+                }
+
+                if ($operationKey === 'update') {
+                    collect($values)->each(function ($singleValues) use ($relation): void {
+                        self::executeUpdate($relation->getModel()->newInstance(), collect($singleValues), $relation);
+                    });
+                }
+
+                if ($operationKey === 'delete') {
+                    $relation->getModel()::destroy($values);
+                }
+            });
+        });
+
+        $belongsToMany->each(function (array $nestedOperations, string $relationName) use ($model): void {
+            /** @var \Illuminate\Database\Eloquent\Relations\BelongsToMany $relation */
+            $relation = $model->{$relationName}();
+
+            collect($nestedOperations)->each(function ($values, string $operationKey) use ($relation): void {
+                if ($operationKey === 'create') {
+                    self::handleMultiRelationCreate(collect($values), $relation);
+                }
+
+                if ($operationKey === 'update') {
+                    collect($values)->each(function ($singleValues) use ($relation): void {
+                        self::executeUpdate($relation->getModel()->newInstance(), collect($singleValues), $relation);
+                    });
+                }
+
+                if ($operationKey === 'delete') {
+                    $relation->getModel()::destroy($values);
+                }
+
```

Comment:
Hi, just pulled your code into my project. 
Could we please also add `belongsToMany.sync()` (more details here @"Syncing Association" https://laravel.com/docs/5.7/eloquent-relationships)

```suggestion
                if ($operationKey === 'sync') {
                    $relation->sync($values);
                }
```

I put this in and it works on my end like expected

Reply:
Agree with this, will probably add this in tommorrow.

Reply:
Done

Reply:
Also, `delete` now also detaches the relation.